master/buildbot/reporters/mail.py

```diff
@@ -43,6 +43,11 @@
 from buildbot.reporters.message import MessageFormatter as DefaultMessageFormatter
 from buildbot.util import service
 
+if PY3:
```

Comment:
@edschofield Someone suggested on IRC to pull you in this discussion as a python3 transition expert. Would you like to advise us on this topic?